Server/ConcordiaCurriculumManager/Program.cs

```diff
@@ -69,14 +69,9 @@ public static void Main(string[] args)
         builder.Services.AddDbContext<CCMDbContext>(options =>
         {
             var dataSourceBuilder = new NpgsqlDataSourceBuilder(dbSetting!.ConnectionString);
-            dataSourceBuilder.MapEnum<RoleEnum>();
```

Comment:
Out of curiosity, why did you remove this line?

Reply:
This was mainly used to create a database enum similar to the RoleEnum. Thus, when you view the table in the database, you won't see integers but the role as an actual string (actually an enum value). I removed it since the other enums were not added in the same way, resulting in some discrepancies. Also, it is easier to seed an integer than an enum in the seeder script.